src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/UsersRepository.cs

```diff
@@ -21,5 +21,16 @@ public async Task<IEnumerable<ApplicationUser>> GetUsersByOrganisationIdAsync(Gu
         {
             return await _context.Users.Where(x => x.PrimaryOrganisationId == organisationId).ToListAsync();
         }
+
+        public async Task CreateUserAsync(ApplicationUser user)
+        {
+            if (user is null)
```

Comment:
user ?? throw

Reply:
As mentioned. This syntax does not work.